install_config/topics/configuring_azure_load_balancer.adoc

```diff
@@ -0,0 +1,117 @@
+balancer in Microsoft Azure and creates the proper firewall rules.
+
+[IMPORTANT]
+====
+Currently, a bug causes extra variables to be included in the Microsoft
+Azure cloud provider when enabling external load balancers. See the
+following for more information:
+
+* https://bugzilla.redhat.com/show_bug.cgi?id=1613546
+====
+
+.Prerequisites
+
+The following items must be added to the
+xref:../install_config/configuring_azure.adoc#azure-configuration-file[the Azure configuration file] located at *_/etc/origin/cloudprovider/azure.conf_*.
+
+Once the values are added, restart the {product-title} services on all hosts:
+
+[source,bash]
+----
+# systemctl restart atomic-openshift-node
+# master-restart api
+# master-restart controllers
+----
+
+== Deploying a sample application using a load balancer
+
+
+.Procedure
+
+. Create a new application:
++
+[source,bash]
+----
+$ oc new-app openshift/hello-openshift
+----
+
+. Expose the load balancer service:
++
+----
+$ oc expose dc hello-openshift --name='hello-openshift-external' --type='LoadBalancer'
+----
++
+This creates a `Loadbalancer` service similar to the following:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: hello-openshift
+  name: hello-openshift-external
+spec:
+  externalTrafficPolicy: Cluster
+  ports:
+  - name: port-1
+    nodePort: 30714
+    port: 8080
+    protocol: TCP
+    targetPort: 8080
+  - name: port-2
+    nodePort: 30122
+    port: 8888
+    protocol: TCP
+    targetPort: 8888
+  selector:
+    app: hello-openshift
+    deploymentconfig: hello-openshift
+  sessionAffinity: None
+  type: LoadBalancer
+----
+
+. To verify that the service has been created:
++
+----
+$ oc get svc
+NAME                       TYPE           CLUSTER-IP       EXTERNAL-IP     PORT(S)                         AGE
+hello-openshift            ClusterIP      172.30.223.255   <none>          8080/TCP,8888/TCP               1m
+hello-openshift-external   LoadBalancer   172.30.99.54     40.121.42.180   8080:30714/TCP,8888:30122/TCP   4m
+----
++
+The `LoadBalancer` type and `External-IP` fields indicate that the service is
+using Microsoft Azure load balancers to expose the application.
+
+This creates the following required objects in the Azure infrastructure:
+
+* A load balancer:
++
+[source,bash]
+----
+az network lb list -o table
+Location    Name         ProvisioningState    ResourceGroup    ResourceGuid
+----------  -----------  -------------------  ---------------  ------------------------------------
+eastus      kubernetes   Succeeded            refarch-azr      30ec1980-b7f5-407e-aa4f-e570f06f168d
+eastus      OcpMasterLB  Succeeded            refarch-azr      acb537b2-8a1a-45d2-aae1-ea9eabfaea4a
+eastus      OcpRouterLB  Succeeded            refarch-azr      39087c4c-a5dc-457e-a5e6-b25359244422
+----
+
+To verify that the load balancer is properly configured, run the following from an external host:
+
+[source,bash]
+----
+$ curl 40.121.42.180:8080
```

Comment:
Are the values here an example, or are they what they are supposed to run? Should I be putting in a replaceable value so they know to replace them? Where do the numbers come from?

Reply:
It may be worth calling them replaceable values. The IP curled is from line 91 external-ip